.circleci/config.yml

```diff
@@ -91,6 +91,7 @@ jobs:
   node4:
     docker:
       - image: node:4
+        user: node
```

Comment:
Why this change? It is unusual to run things in Docker not as root.

Reply:
To save 5 minutes per CI iteration.